python_modules/dagster/dagster/grpc/server.py

```diff
@@ -812,6 +812,10 @@ def __init__(
         self.server = grpc.server(
             ThreadPoolExecutor(max_workers=max_workers),
             compression=grpc.Compression.Gzip,
+            options=[
+                ("grpc.max_send_message_length", max_send_bytes()),
+                ("grpc.max_receive_message_length", max_rx_bytes()),
+            ],
```

Comment:
nit: this seems like it could be pulled out into a separate diff.

Reply:
yeah, fair - this was actually to fix one of the scheduler tests that was failing to fetch the execution plan b/c it was too large (now that we are throwing instead of creating a failed run).